rollback_setup.sh

```diff
@@ -7,17 +7,36 @@ while true; do
     esac
 done
 
-# Uninstall Leapp CLI
-brew uninstall Noovolari/brew/leapp-cli
-# Uninstall Session Manager Plugin
-brew uninstall --cask session-manager-plugin
-# Uninstall AWS CLI
-brew uninstall awscli
-# Remove AWS credential files
-rm -rf ~/.aws
-# Uninstall python
-brew uninstall python --ignore-dependencies python
-# Uninstall homebrew
-sudo /bin/bash -cf "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall.sh)"
-# Uninstall the Xcode CLT (this may be installed as part of homebrew)
-sudo rm -rf /Library/Developer/CommandLineTools
+kernel_name=$(uname -s)
+
+if [[ "$kernel_name" == 'Darwin' ]]; then
+    # Uninstall Leapp CLI
+    brew uninstall Noovolari/brew/leapp-cli
+    # Uninstall Session Manager Plugin
+    brew uninstall --cask session-manager-plugin
+    # Uninstall AWS CLI
+    brew uninstall awscli
+    # Remove AWS credential files
+    rm -rf ~/.aws
+    # Uninstall python
+    brew uninstall python --ignore-dependencies python
+    # Uninstall homebrew
+    sudo /bin/bash -cf "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall.sh)"
+    # Uninstall the Xcode CLT (this may be installed as part of homebrew)
+    sudo rm -rf /Library/Developer/CommandLineTools
+elif [[ "$kernel_name" == 'Linux' ]]; then
+    # Uninstall Leapp CLI
+    sudo npm uninstall -g @noovolari/leapp-cli
+    # Remove node
+    sudo apt remove -y npm
+    sudo apt remove -y nodejs
+    # Remove leapp
+    sudo dpkg -r leapp
+    sudo dpkg -P leapp
+    # Remove session-manager-plugin
+    sudo dpkg -r session-manager-plugin
+    sudo dpkg -P session-manager-plugin
```

Comment:
Can you combine the remove and purge flags to run this once for each package?

Reply:
No, dpkg complains if you do